api/docs/v2/storefront/index.yaml

```diff
@@ -63,6 +63,16 @@ paths:
                     password_confirmation:
                       type: string
                       example: spree123
+                    public_metadata:
+                      type: object
+                      example:
+                        number_of_orders: 0
+                      description: The public metadata for this User
+                    private_metadata:
+                      type: object
+                      example:
+                        has_abandoned_cart: false
+                      description: The private metadata for this User
```

Comment:
```suggestion
```

Reply:
This is for the parameter when creating the Account. You set the private metadata by passing the private_metadata param. Then in the response I have changed it to only show the public metadata